specs/core/0_beacon-chain.md

```diff
@@ -967,17 +990,19 @@ def get_effective_balance(validator: ValidatorRecord) -> int:
 
 ```python
 def get_new_validator_registry_delta_chain_tip(current_validator_registry_delta_chain_tip: Hash32,
-                                               index: int,
+                                               validator_index: int,
```

Comment:
Also, need to update the keyword argument where that this function is being called.
(in `def activate_validator()` and in `def exit_validator()`)